tests/lib/rules/no-invalid-resource-location.js

```diff
@@ -36,20 +36,26 @@ ruleTesterInstance.run('no-invalid-resource-location', ruleUnderTest, {
   ],
```

Comment:
I think it makes sense to add more positive/negative cases for partials (in layouts, modules components).

Reply:
Nice catch!